examples/hooks/App.tsx

```diff
@@ -56,7 +62,16 @@ export function App() {
           />
         </div>
         <div style={{ display: 'grid', gap: '.5rem' }}>
-          <SearchBox placeholder="Search" />
+          <div style={{ display: 'flex' }}>
```

Comment:
this style probably should be in the css too for consistency? it's a little odd how it's mixed now